dashboard/dashboard.js

```diff
@@ -446,9 +446,9 @@ function populateInfoBoxes (singleBuildingData,categoryData,floorAreaRange) {
   )
   d3.selectAll('.building-type-sq-ft').text(numberWithCommas(floorAreaRange[0]) + '-' + numberWithCommas(floorAreaRange[1]))
 
-  let euirank = rankBuildings(singleBuildingData.ID, categoryData, 'latest_weather_normalized_site_eui_kbtu_ft2')
-  d3.select('#building-ranking').text(euirank[0])
-  d3.select('#total-building-type').text(euirank[1])
+  let zscorerank = rankBuildings(singleBuildingData.ID, categoryData, 'zscoreVal', 'latest_site_eui_kbtu_ft2')
```

Comment:
we should be ranking first by estar, then looping back to rank by eui where `a.estar === b.estar`, and not worry about calculating zscore